### PR TITLE
feat: fix toolbar on narrow screens

### PR DESCRIPTION
**Pull request by dogfrogfog**
## Brief
- https://github.com/pyroscope-io/pyroscope/issues/1726

## Changes
- add ability to show 2nd toolbar row
- add "more" button to open collapsed toolbar items

### CODE REVIEW

File: packages/pyroscope-flamegraph/src/Toolbar.tsx [hunk 1]
```diff
@@ -33,8 +42,8 @@ export const TOOLBAR_MODE_WIDTH_THRESHOLD = 900;
 
 export type ShowModeType = ReturnType<typeof useSizeMode>;
 
-export const useSizeMode = (target: React.RefObject<HTMLDivElement>) => {
-  const [size, setSize] = React.useState<'large' | 'small'>('large');
+export const useSizeMode = (target: RefObject<HTMLDivElement>) => {
```
Comment (dogfrogfog) [inline packages/pyroscope-flamegraph/src/Toolbar.tsx:45]: also guys what do you think about this: if we have this "more" button functionality we can go towards removing `size` logic, where we have 

large
Screenshot 2022-11-23 at 11 11 17

and small
Screenshot 2022-11-23 at 11 11 40

toolbars depending on width and just show **large** option with additional "more" button (3dot) 

cc @eh-am @Rperry2174

Comment (eh-am) [reply packages/pyroscope-flamegraph/src/Toolbar.tsx:45]: I agree, that simplifies a lot!

**Comment by dogfrogfog** [comment]
https://user-images.githubusercontent.com/47758224/204330962-f340fb96-ffe0-404c-87e8-9d596d4c0174.mov

**Comment by Rperry2174** [comment]
Looks great so far! dropdown gets hidden beneath the flamegraph: 

https://user-images.githubusercontent.com/23323466/204337780-b3f981ef-39b0-415b-acd6-3c0c8c232ff5.mov

**Comment by Rperry2174** [comment]
Also we currently drop groups starting on the right and moving to the left ... How hard would it be to reverse that so that the first items that disappear are the items on the _left_ (i.e. head-first / tail first would disappear first)

**Comment by dogfrogfog** [comment]
Screenshot 2022-11-29 at 12 00 54

**Comment by dogfrogfog** [comment]
https://user-images.githubusercontent.com/47758224/204531075-fa6990e1-94d8-4f65-953f-0f93e67c74be.mov

**Review by Rperry2174 — APPROVED** [review]
love it! Thanks @dogfrogfog

**Pull request merged**